docs/VSExtension/README.md

```diff
@@ -98,9 +98,9 @@ A list of changes can be found in the [Changelog](Changelog.md).
 [BinaryAnalysis]: img/analyzeToolbar.png
 [OptionsPane]: img/optionsPanel.png
 [PlatformPortability]: ../HowTo/PlatformPortability.md
-[PrivacyPolicy]: ../LicenseTerms/Microsoft%20.NET%20Portability%20Analyzer%20Privacy%20Statement.txt
+[PrivacyPolicy]: ../LicenseTerms/Microsoft%20.NET%20Portability%20Analyzer%20CLI%20Privacy%20Statement.txt
 [ReportToolWindow]: img/report.toolWindow.png
 [SampleReport]: img/analysisReport.png
 [SolutionExplorer-ContextMenu]: img/analysisContextMenu.png
 [SourceCodeMapping]: img/sourceMapping.output.png
-[VSGallery]: https://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b
\ No newline at end of file
+[VSGallery]: https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer
```

Comment:
Is this released under your name?

Reply:
Yes. It's because it had to belong to a personal MSDN account and we don't have a team account. So it was moved to me.